plays/utn-deploy.yml

```diff
@@ -0,0 +1,15 @@
+- name: Deploy tb-website-builds files to web servers
+  hosts: "{{ var_hosts | default('webheads') }}"
+  vars_files:
+  - ../vars/conf.yml
+
+  tasks:
+    - name: checkout tb-website-builds to get the start page files
+      git:
+        repo: "https://github.com/thunderbird/tb-website-builds.git"
+        dest: "{{ utn_files }}"
+        version: "{{ branch | default('prod') }}"
+        force: True
+
+    - name: write the commit hash to version.txt
+      shell: 'cd "{{ utn_files }}" && git rev-parse HEAD > site/version.txt'
```

Comment:
I guess I could bundle this into start-deploy and avoid having to update the webhook. Any thoughts @Sancus ?

Reply:
Yeah there should just be one script since it's just dumping all the contents of `tb-website-builds` into the right place in both cases.

Some stuff here is misnamed because the 'start' folder actually contains both the start page and the main website, which is definitely confusing.